### ISSUE TITLE
Bugged potions

### ISSUE THREAD

**Issue by Rahizm** [dalaranwow/dalaran-wow#5806]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Health, mana, guardian, and battle potions and elixers have a high chance to be unusable. The button will NOT be greyed out but when used, an error will be given the same as if you were using a potion on cooldown. It is unknown what causes this bug as it occurs randomly. The only workaround I have been able to find is to log out with missing health and log back in. When I tested on my level 20 warlock, the bug persisted through death, logging out and back in with full health, dismissing and summoning pet, changing characters, and hearthing. This bug also prevents the drinking of elixers that have no cooldown. This bug does NOT prevent the use of healthstones.

**Expected behaviour**: Tell us what should happen instead
All health and mana potions should be start a one (1) minute cooldown when consumed and be able to used freely when not on cooldown. Guardian and battle elixers should not have a cooldown associated with them and should be able to be used at any time.

**Steps to reproduce the problem**:

1. Unable to determine. Bug occurred at random in multiple, varied scenarios.
2. 
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**
